packages/orchestration/src/fetched-chain-info.js

```diff
@@ -1,5 +1,141 @@
 /** @file Generated by fetch-chain-info.ts */
 export default {
+  agoric: {
+    chainId: 'agoric-3',
+    stakingTokens: [
+      {
+        denom: 'ubld',
+      },
+    ],
```

Comment:
Tangential to the PR, but i think this should be `bondDenom: string` instead of an arary.